File: ex1.c
/*Project Euler 1 in c*/
#include <stdio.h>

int main(){
	printf("Sum: %i\n", run(1000));
	return 0;
}
int run(int limit){
	int x;
	int sum =0;
	for(x=1; x<limit; x++){
		if(x%3==0){
			sum=sum+x;
			continue;
		}
		if(x%5==0){
			sum=sum+x;
			continue;
		}
		}
		return sum;
		}

File: ex1.py
#Project Euler 1
#19/08/13

#"If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23. Find the sum of all the multiples of 3 or 5 below 1000."

#Initial longer solution
# multiples=[]
# for i in range(0,1000):
# 	if i%3==0 or i%5==0 and i not in multiples:
# 		multiples.append(i)
# print(sum(multiples))



#Second shorter lambda solution
print(sum(i for i in range(1, 1000) if i%3==0 or i%5==0))
